ui/app/AppLayouts/Wallet/views/SavedAddressesView.qml

```diff
@@ -81,46 +83,87 @@ Item {
         StatusListItem {
             id: savedAddress
             title: name
-            subTitle: address
-            icon.name: "wallet"
+            subTitle: name + " \u2022 " + Utils.getElidedCompressedPk(address)
             implicitWidth: parent.width
+            color: "transparent"
+            border.color: Theme.palette.baseColor5
+            //titleTextIcon: RootStore.favouriteAddress ? "star-icon" : ""
+            statusListItemComponentsSlot.spacing: 0
             property bool showButtons: sensor.containsMouse
+
             components: [
                 StatusRoundButton {
-                    color: hovered ? Theme.palette.dangerColor2 : Theme.palette.dangerColor3
-                    icon.color: Theme.palette.dangerColor1
-                    visible: showButtons
-                    icon.name: "delete"
+                    icon.color: savedAddress.showButtons ? Theme.palette.directColor1 : Theme.palette.baseColor1
+                    type: StatusRoundButton.Type.Tertiary
+                    icon.name: "send"
                     onClicked: {
-                        deleteAddressConfirm.name = name
-                        deleteAddressConfirm.address = address
-                        deleteAddressConfirm.open()
+                        root.sendModal.open(address);
                     }
                 },
-                StatusRoundButton {
-                    icon.name: "pencil"
-                    visible: showButtons
-                    onClicked: Global.openPopup(addEditSavedAddress,
-                                                {
-                                                    edit: true,
-                                                    address: address,
-                                                    name: name
-                                                })
+                CopyToClipBoardButton {
+                    type: StatusRoundButton.Type.Tertiary
+                    icon.color: savedAddress.showButtons ? Theme.palette.directColor1 : Theme.palette.baseColor1
+                    store: RootStore
+                    textToCopy: address
                 },
+//                StatusRoundButton {
```

Comment:
cab we add a comment saying to-do ?

Reply:
@Khushboo-dev-cpp issue is here https://github.com/status-im/status-desktop/issues/6546, TODO also added